HoneyBadgers.Logic/Enums/MovieStatusVIew.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace HoneyBadgers.Logic.Enums
+{
+    public enum MovieStatusView
+    {
+        [Display(Name = "watched")]
+        Watched,
+        [Display(Name = "want to watch")]
+        WantToWatch,
+        [Display(Name = "don't want to watch it")]
+        Ignore
```

Comment:
To jest fajny pomysł. W przyszłości można pozwolić uzytkownikowi na ignorowanie newsów/recenzji jakiegoś filmu albo jakiś takich rzeczy ;)

Reply:
właśnie wiem na jakieś stronie z filmami zobaczyłam coś podobnego